src/ext/gui.rs

```diff
@@ -39,6 +53,11 @@ pub struct clap_plugin_gui {
         api: *const c_char,
         is_floating: bool,
     ) -> bool,
+    pub get_preferred_api: unsafe extern "C" fn(
+        plugin: *const clap_plugin,
+        api: *const c_char,
+        is_floating: bool,
```

Comment:
I believe these should be `*mut *const c_char` and `*mut bool`: https://github.com/free-audio/clap/blob/045eeffda9e777aacf189c8325b505bc8773ad82/include/clap/ext/gui.h#L100

Reply:
You're right, fixed.